core/domain/beam_job_validators.py

```diff
@@ -0,0 +1,88 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Validators for Apache Beam job models."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.domain import base_model_validators
+from core.platform import models
+from jobs import registry as jobs_registry
```

Comment:
The `jobs_registry` name is bit confusing as we have another file core/jobs_registry.py and while reading line 39 I assumed jobs_registry to be the core.jobs_registry module.

Reply:
I think that's OK since we'll be deleting that file as part of the Python 3 migration.